tracer/src/Datadog.Trace/Agent/MessagePack/MessagePackStringCache.cs

```diff
@@ -34,6 +34,7 @@ internal static class MessagePackStringCache
 
     // NOTE: all of these can be cached in SpanMessagePackFormatter as static byte[]
     // since they never change over the lifetime of a process
+    private static CachedBytes _processTags;
```

Comment:
See the comment above 😅 

> NOTE: all of these can be cached in SpanMessagePackFormatter as `static byte[]` since they never change over the lifetime of a process

See `_languageValueBytes` and `_runtimeIdValueBytes` in `SpanMessagePackFormatter` for examples.

Reply:
we can fix that in a different PR I guess